lib/usd/ui/layerEditor/stageSelectorWidget.h

```diff
@@ -39,12 +39,13 @@ class StageSelectorWidget : public QWidget
     StageSelectorWidget(SessionState* in_sessionState, QWidget* in_parent);
 
 protected:
-    void                setSessionState(SessionState* in_sessionState);
-    pxr::UsdStageRefPtr selectedStage();
+    void                   setSessionState(SessionState* in_sessionState);
+    PXR_NS::UsdStageRefPtr selectedStage();
 
     // slot:
-    void updateFromSessionState(pxr::UsdStageRefPtr const& stageToSelect = pxr::UsdStageRefPtr());
-    void stageRenamed(std::string const& name, pxr::UsdStageRefPtr const& stage);
+    void
+         updateFromSessionState(PXR_NS::UsdStageRefPtr const& stageToSelect = PXR_NS::UsdStageRefPtr());
```

Comment:
An unfortunate choice made by `clang-format` here. :(

Reply:
We keep track of issues like this. If you have to know the configuration that would prevent this, please send it my way and I will put it in our internal wiki. Once we will be considering adopting clang-format v11, we will revisit configuration settings to consider tweaks for annoying parts people are finding.

Reply:
Thanks @kxl-adsk! Looking at the [current set of options](https://clang.llvm.org/docs/ClangFormatStyleOptions.html), I think it might be `IndentWrappedFunctionNames: 'false'`? I haven't had a chance to build a newer version of clang-format to try it out though. Looks like it was known as `IndentFunctionDeclarationAfterType` for a while too: https://github.com/llvm/llvm-project/commit/c75e1effed73e1a3f7e9b509634648d71c1dddcf